apps/src/templates/sessions/LockoutPanel.jsx

```diff
@@ -6,8 +6,8 @@ import {getStore} from '../../redux';
 import {isEmail} from '@cdo/apps/util/formatValidation';
 import cookies from 'js-cookie';
 import * as color from '../../util/color';
-import headerImage from './images/lockout_penguin.png';
-import headerThanksImage from './images/dancing_penguin.png';
+import headerImage from '@cdo/static/common_images/penguin/yelling.png';
+import headerThanksImage from '@cdo/static/common_images/penguin/dancing.png';
```

Comment:
I'm very pro this change.

Reply:
It makes sense to have the images be near the components that they are used uniquely within and then update them when the images are used in a second container by moving them to the common path.

I am considering namespacing this path so is at least slightly clearer what the context of these images are, but maybe that's not too necessary. A common path just becomes a swamp of random images... which might be nice to avoid. Not a very important problem, though.